docs/docs/spec.md

```diff
@@ -258,11 +267,34 @@ Chunk files can be:
 
 Applications may choose to arrange chunks within files in different ways to optimize I/O patterns.
 
+### Transaction logs
+
+Transaction logs keep track of the operations done in a commit. They are not used to read objects
+from the repo, but they are useful for features such as commit diff and conflict resolution.
+
+Transaction logs are an optimization, to provide fast conflict resolution and commit diff. They are
+not absolutely required to implement the core Icechunk operations.
+
+Transaction log files are encoded using [flatbuffers](https://github.com/google/flatbuffers). The IDL for the
+on-disk format can be found in [the repository file](https://github.com/earth-mover/icechunk/tree/main/icechunk/flatbuffers/transaction_log.fbs)
```

Comment:
We should explicitly link to every flatbuffers file like this if not already done.

Reply:
Yes, this was done some time ago